index.rst

```diff
@@ -28,7 +30,7 @@ Selected datasets, including both raw and prompt-processed, will be copied from
 
 Removal from the embargo repo can occur after a successful copy to normal storage, or it can be postponed for a configurable time period, but space limitations mean that no dataset should be expected to be retained for more than 30 days in the embargo repo.
 
-Handling of OUO Satellite Catalog
+Handling of the Satellite Catalog
 =================================
 
-The catalog will be treated as CUI Basic, per DOE Order 471.7. As such, access will be restricted to authorized individuals, identified in an approved acccess list and allowed access to the catalog via permissions set on a directory in the emabargo rack filesystem. The catalogs may not leave that directory, except for transient copies needed by prompt/alert processing.
+The catalog will be treated as CUI Basic, per DOE Order 471.7. As such, access will be restricted to authorized individuals, identified in an approved acccess list and allowed access to the catalog via permissions set on a directory in the embargo rack filesystem. The catalogs may not leave that directory, except for transient copies needed by prompt/alert processing. No derivative catalogs may be stored.
```

Comment:
I worry this prevents us coming up with a temporary catalog and keeping it on embargo .. I think you mean no derivative maybe stored outside embargo rack.

Reply:
Not from my reading. They say we have to treat any such catalogue as OUO... we're trying to avoid that...

Reply:
Correct. I'm investigating whether tables of performance metrics count as "derivative catalogs" - I don't think they should.

Reply:
Let's keep the line about no derivative catalogs. John Sarrao confirmed that tabulating metrics is OK, and this is what @ebellm needs most, IIUC.